cpp/pagerank_module/pagerank_online_module.cpp

```diff
@@ -111,9 +120,18 @@ void OnlinePageRankUpdate(mgp_list *args, mgp_graph *memgraph_graph, mgp_result
     }
 
     for (auto const &[node_id, rank] : pageranks) {
-      auto record = record_factory.NewRecord();
-      record.Insert(kFieldNode, graph.GetNodeById(mgp::Id::FromUint(node_id)));
-      record.Insert(kFieldRank, rank);
+      // As IN_MEMORY_ANALYTICAL doesn’t offer ACID guarantees, check if the graph elements in the result exist
+      try {
+        // If so, throw an exception:
+        const auto maybe_node = graph.GetNodeById(mgp::Id::FromUint(node_id));
```

Comment:
The same comment as in `katz_centrality_online_module.cpp`

Reply:
Done